frontend/javascripts/viewer/model/accessors/disabled_tool_accessor.ts

```diff
@@ -83,9 +92,21 @@ const getExplanationForDisabledVolume = (
 const ALWAYS_ENABLED_TOOL_INFOS = {
   [AnnotationTool.MOVE.id]: NOT_DISABLED_INFO,
   [AnnotationTool.LINE_MEASUREMENT.id]: NOT_DISABLED_INFO,
-  [AnnotationTool.AREA_MEASUREMENT.id]: NOT_DISABLED_INFO,
 };
 
+function _getAreaMeasurementToolInfo(isFlycamRotated: boolean) {
+  return {
+    [AnnotationTool.AREA_MEASUREMENT.id]: isFlycamRotated
+      ? {
+          isDisabled: true,
+          explanation: rotationActiveDisabledExplanation,
+        }
+      : NOT_DISABLED_INFO,
+  };
+}
```

Comment:
I disabled the area measurement tool during rotated state for now as getting the calculation correct seemed a little more complicated and I did not want to implement this in the first iteration. I imagined that this tool would not be that important in a rotated state for now.